packages/lodestar-cli/src/networks/index.ts

```diff
@@ -2,11 +2,12 @@ import got from "got";
 import {IBeaconNodeOptions} from "@chainsafe/lodestar";
 import {RecursivePartial} from "@chainsafe/lodestar-utils";
 import {IBeaconParamsUnparsed} from "../config/types";
-import * as pyrmont from "./pyrmont";
 import * as mainnet from "./mainnet";
+import * as pyrmont from "./pyrmont";
+import * as prater from "./prater";
 
-export type NetworkName = "pyrmont" | "mainnet";
-export const networkNames: NetworkName[] = ["pyrmont", "mainnet"];
+export type NetworkName = "mainnet" | "pyrmont" | "prater" | "dev";
+export const networkNames: NetworkName[] = ["mainnet", "pyrmont", "prater"];
```

Comment:
Consider rebasing the PR at origin/master to not show unnecessary diffs

Reply:
done